assembl/static/js/app/views/preferencesView.js

```diff
@@ -523,7 +523,7 @@ var UrlPreferenceView = StringPreferenceView.extend({
   },
   regexp: new RegExp('^(?:(?:http|https|ftp)://)(?:\\S+(?::\\S*)?@)?(?:(?:(?:[1-9]\\d?|1\\d\\d|2[01]\\d|22[0-3])(?:\\.(?:1?\\d{1,2}|2[0-4]\\d|25[0-5])){2}(?:\\.(?:[0-9]\\d?|1\\d\\d|2[0-4]\\d|25[0-4]))|(?:(?:[a-z\\u00a1-\\uffff0-9]+-?)*[a-z\\u00a1-\\uffff0-9]+)(?:\\.(?:[a-z\\u00a1-\\uffff0-9]+-?)*[a-z\\u00a1-\\uffff0-9]+)*(?:\\.(?:[a-z\\u00a1-\\uffff]{2,})))|localhost)(?::\\d{2,5})?(?:(/|\\?|#)[^\\s]*)?$', 'i'),
   processValue: function(value) {
-    if (!this.regexp.test(value)) {
+    if (!this.regexp.test(value.trim())) {
         throw i18n.gettext("This does not appear to be a URL");
```

Comment:
should we trim the value outside the test? (ie before assignment.) It might make sense in many cases (url, email, int...) and maybe even all cases.